src/app/splashscreen/splashscreen.component.scss

```diff
@@ -0,0 +1,22 @@
+@import “../variables”;
+
+.logo{
+  max-width: 20%;
+}
+
+.titleleft{
```

Comment:
Its again the same, the same heading on splash screen, login page and registration page. Will make the name same, it was different because we  were working separately on it.